fieldservice/views/menu.xml

```diff
@@ -50,11 +50,11 @@
               parent="operations"
               sequence="30"/>
 
-    <menuitem id="menu_fsm_route"
+    <!--menuitem id="menu_fsm_route"
               name="Day Routes"
               action="action_fsm_route"
               parent="operations"
-              sequence="40"/>
+              sequence="40"/-->
```

Comment:
@osi-scampbell Please rebase. This menu item is now only shown in the developer mode. No need to comment it.

Reply:
@osi-scampbell My bad. It will come with #215. Please remove the comments for now.